mungegithub/submit-queue/pv.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  labels:
+    app: submit-queue
+  name: submit-queue-github-cache
+spec:
+  capacity:
+    storage: 1Gi
```

Comment:
Is it really 10 Gi or is it 10 G ? /me hates storage and their non-base2 units.  Doesn't really matter.